RELEASING.md

```diff
@@ -1,12 +1,8 @@
-###Overview
-
-Most of the release process is handled by the uploadArchives task. The task can be triggered by
-running "./gradlew uploadArchives" from the project's directory. A new artifact will be uploaded
-to the staging repository in Sonatype when "-SNAPSHOT" is not included in the version (as
-listed in the base directory's `pom.xml`). When "-SNAPSHOT" is included, the task only updates the
-artifact in the snapshot repository.
-
-###Release Instructions
+Versioning
```

Comment:
This comment seems to just be floating about with no context. Based on the context in the version prior to your change, I don't think this section is necessary at all.